src/reader.conf.in

```diff
@@ -7,6 +7,7 @@
 #     GemCoreSIMPro2
 #     GemPCPinPad
 #     GemPCTwin (default value)
+#     SEC1210 (Dual Uart Reader)
```

Comment:
I guess you mean "dual slot" here?
The way the two slots are internally managed is not really relevant for users.